src/resources/Strings.tsx

```diff
@@ -128,6 +128,10 @@ export const Strings = {
         text: 'All revisions',
       },
     },
+    searchResultsInput: {
+      placeholder: 'title, platform, revision or options',
```

Comment:
I just did that (increased the width), and replaced by "Search results" like Dasha suggested

Reply:
BTW I've put the width to 50% instead of a precise px value.
Ideally we should use MUI grids for this (https://mui.com/material-ui/react-grid2/) but I didn't want to change this now... but this could make it easier for us to support smaller screens, where we're not great at at the moment.